src/main/java/com/ita/if103java/ims/security/UserDetailsImpl.java

```diff
@@ -29,7 +32,15 @@ public UserDetailsImpl() {
 
     @Override
     public Collection<? extends GrantedAuthority> getAuthorities() {
-        return Collections.singletonList(new SimpleGrantedAuthority(user.getRole().name()));
+        if (!user.isActive()) {
+            return Collections.emptyList();
+        }
```

Comment:
is it useful? in this class, there is a method `isEnabled()` that indicates whether a user is active

Reply:
Never, good that you've noticed